pyccel/ast/core.py

```diff
@@ -248,27 +247,46 @@ class AsName(Basic):
 
     Parameters
     ==========
-    name   : str
-             original name of variable or function
+    obj    : Basic or BasicType
+             The variable, function, or module being renamed
```

Comment:
Could you please add an example to the docstring where `obj` is a `type`?

Reply:
Fixed d0cbebc3

Reply:
Thanks!